### DISCUSSION TITLE
order of current_zones and entered_zones

### DISCUSSION THREAD

**Discussion by vajonam** [blakeblackshear/frigate#6189] [Ask A Question]
does the order of the zone names in the entered_zones  match to chronological order? so index 0 is first zone the object entered? 

Based on this it appears to get appended, https://github.com/blakeblackshear/frigate/blob/8535213ad96ff34b85b306e0f587afdada216ccb/frigate/object_processing.py#L152 just want to verify this. 

Assuming zones dont over lap and objects have only a few paths to get from zone 1 to 2 to 3, will this assumption hold true? 

reason I am asking is am looking to make my notifications a bit more advanced and to be able to determine how an object (car) came in the driveway for example from the garage or from the road.

**Comment by NickM-27** [comment]
Yes, they are chronological order
